tests/unit/plugins/modules/net_tools/test_nmcli.py

```diff
@@ -1221,9 +1221,8 @@
 connection.permissions:                 brittany
 ipv4.method:                            auto
 ipv6.method:                            auto
-vpn-type:                               l2tp
 vpn.service-type:                       org.freedesktop.NetworkManager.l2tp
-vpn.data:                               gateway=vpn.example.com, password-flags=2, user=brittany, ipsec-enabled=true, ipsec-psk=QnJpdHRhbnkxMjM=
+vpn.data:                               gateway = vpn.example.com, ipsec-enabled = true, ipsec-psk = QnJpdHRhbnkxMjM=, password-flags = 2, user = brittany
```

Comment:
Maybe also keep some data with no space, or multiple spaces?

Reply:
That's fine. I removed spaces from the pptp test case.